### ISSUE TITLE
inject html displays for zeppelin note coming from databricks scala notebook with displayHTML function

### ISSUE THREAD

**Issue by lamastex** [TiloWiklund/pinot#5]
%spark
///////////////////////////////////////////////////////////////////////////////////////////////////////////
// this snippet needs to be injected into first cell of a Zeppelin note that is created from a Scala 
// databricks notebook with 'displayHTML' --- this renders html in the zeppelin note coming from databricks 
///////////////////////////////////////////////////////////////////////////////////////////////////////////
def displayHTML(htmlInput:String):  Unit = {
    println("%html")
    println(htmlInput)
}
///////////////////////////////////////////////////////////////////////////////////////////////////////////

**Comment by TiloWiklund** [comment]
I think the best way to do this is to write a small utility that adds cells to a notebook (which can then be run separately on pinot's output). pinot does not currently attempt to define or track anything like an "origin format" of a notebook.

**Comment by lamastex** [comment]
Yes, this is a good idea. Would you be able to do such an injection for zeppelin notebooks coming from databricks? Then we will be done with the databricks to zeppelin pipeline for notebooks. We may need a similar injection for jupyter ipynb notebooks from databricks too.
